app/views/application/_footer.haml

```diff
@@ -11,6 +11,9 @@
       %li
         Read our blog
         = link_to "openaustraliafoundation.org.au/blog", "http://www.openaustraliafoundation.org.au/blog"
+      %li
+        Fork this project
```

Comment:
While that's true, its the github link - wouldn't the intended users be developers?

How does just "Github" or "Us on Github" work for you?